.gitignore

```diff
@@ -214,6 +214,12 @@ PublishScripts/
 *.nuget.props
 *.nuget.targets
 
+# Vcpkg Packages
+**/vcpkg_installed/*
+# when building using Nuke and vcpkg is not installed (or in the PATH)
+# the target SetupVcpkg will install it at the root of the repo
```

Comment:
Just a nit, but can we control these paths? _In general_ we're moving towards the "new" [artifacts layout](https://learn.microsoft.com/en-us/dotnet/core/sdk/artifacts-output). i.e. everything goes in the `artifacts` folder, with a sub folder for "category" e.g. currently we have

```
artifacts
 |___ bin
       |___ Project1
       |___ Project2
 |___ obj
       |___ Project1
       |___ Project2
 |___ publish 
       |___ Project1
       |___ Project2
 |___ build_data
       |___ logs
       |___ results
```

It would be nice if we could have (for example) a `deps` or `dependencies` folder, with a vcpkg subfolder perhaps? Something like that?

Currently we're only putting the samples and things in `artifacts` but ultimately I'd like to get _everything_ building there, as it will give us some CI wins apart from anything else 🙂

Reply:
good idea. I'll get to it.

Reply:
for this specific one, it might be tricky but I'll check